### PR TITLE
Jhbate/metrics

### PR DESCRIPTION
**Pull request by jh-bate**
@jebeck this add's some of the missing metrics calls that @brandonarbiter requested. I have tested localhost - there should be no real surprises.

### CODE REVIEW

File: app/components/patientcard/patientcard.js [hunk 1]
```diff
@@ -85,9 +85,19 @@ var PatientCard = React.createClass({
       'patientcard-actions--highlight': (!this.props.isNavbar && this.state.highlight === 'view') || this.props.currentPage && this.props.currentPage.match(/(data)$/i)
     });
 
+    var self = this;
+    var handleClick = function(e) {
+      if (e) {
+        e.preventDefault();
+        e.stopPropagation();
+      }
```
Comment (jebeck) [inline app/components/patientcard/patientcard.js]: Why are these here? They prevent following the link...?

File: app/components/patientcard/patientcard.js [hunk 2]
```diff
@@ -172,10 +182,19 @@ var PatientCard = React.createClass({
       'patientcard-actions--highlight': (!this.props.isNavbar && this.state.highlight === 'share')  || this.props.currentPage && this.props.currentPage.match(/(share)$/i)
     });
 
+    var self = this;
+    var handleClick = function(e) {
+      if (e) {
+        e.preventDefault();
+        e.stopPropagation();
+      }
```
Comment (jebeck) [inline app/components/patientcard/patientcard.js]: Ditto.

**Comment by jh-bate** [comment]
@jebeck I have updated the PR

**Comment by jebeck** [comment]
Fixed, but some of the metrics calls appear to be repeated. Is that a problem?

From the logs:
Api              | GET /metrics/Clicked%20VDF%20View%20Data
api.js:580 Api              | GET /metrics/Clicked%20Own%20Care%20Team
api.js:580 Api              | GET /metrics/Clicked%20Own%20Care%20Team

You get the second one twice every time you click View from the VDF page.

**Comment by jh-bate** [comment]
I will have a look. It won't cause an issue other than noise in metrics

**Comment by jebeck** [comment]
OK, then LGTM.
